frontend/src/modules/Traces/TraceFilter.tsx

```diff
@@ -54,7 +60,20 @@ const _TraceFilter = (props: TraceFilterProps) => {
 			tags: [],
 			operation: "",
 			latency: { min: "", max: "" },
+			kind: ""
 		});
+
+		const spanKindList: ISpanKind[] = [
+			{
+				label: "SERVER",
+				value: "2"
+			},
+			{
+				label: "CLIENT",
+				value: "3"
+			}
+		]
+		setSpanKinds(spanKindList);
```

Comment:
Instead of setting state, you can just define a variable and use it in the component. Since the options remains constant across different renders

Reply:
Will do the change